From 8dd29e267a0ea4100aea9c539a723374f3119743 Mon Sep 17 00:00:00 2001
Message-Id: <8dd29e267a0ea4100aea9c539a723374f3119743.1421709247.git.chang-joon.lee@intel.com>
In-Reply-To: <3650eb44734ad4fbe098313349da8185595a20d8.1421709247.git.chang-joon.lee@intel.com>
References: <3650eb44734ad4fbe098313349da8185595a20d8.1421709247.git.chang-joon.lee@intel.com>
From: Shobhit Kumar <shobhit.kumar@intel.com>
Date: Fri, 26 Sep 2014 15:02:16 +0530
Subject: [PATCH 32/75] MUST_REBASE [VPG]: adf/intel: Signal to i915 that ADF
 driver is ready

Calls ADF wrapper function in ADF to set global flag which is checked in
vlv IRQ handler before passing display interrupts to ADF driver

MUST_REBASE: Since Google ADF framework is not in upstream and also
we cannot have two display drivers, other one being i915, we need to
work on i915 and ADF convergence path before it can be upstreamed

For: GMINL-3547
Change-Id: Ie61aa5db94ee3caa07b3f8908765ea511f2d9c45
Signed-off-by: Shobhit Kumar <shobhit.kumar@intel.com>
---
 drivers/video/adf/intel/core/intel_adf_drv.c |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/video/adf/intel/core/intel_adf_drv.c b/drivers/video/adf/intel/core/intel_adf_drv.c
index e839845..0bd77ae 100644
--- a/drivers/video/adf/intel/core/intel_adf_drv.c
+++ b/drivers/video/adf/intel/core/intel_adf_drv.c
@@ -39,6 +39,9 @@ static int intel_adf_init(void)
 		return -EINVAL;
 	}
 
+	/* indiacte to i915 that we are ready */
+	set_adf_ready();
+
 	pr_err("ADF: %s\n", __func__);
 	return 0;
 }
-- 
1.7.9.5

